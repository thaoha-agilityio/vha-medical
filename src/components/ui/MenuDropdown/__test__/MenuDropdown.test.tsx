import { render } from '@testing-library/react';
import { MenuDropdown, MenuDropdownProps } from '..';

describe('MoreAction test cases', () => {
  const setup = (props: MenuDropdownProps) =>
    render(<MenuDropdown {...props} />);

  const mockProps: MenuDropdownProps = {
    options: [
      {
        key: '1',
        label: 'Mock',
        isDisabled: false,
      },
    ],
    icon: <></>,
  };
  it('should render correctly', () => {
    const { asFragment } = setup(mockProps);

    expect(asFragment()).toMatchSnapshot();
  });
});
