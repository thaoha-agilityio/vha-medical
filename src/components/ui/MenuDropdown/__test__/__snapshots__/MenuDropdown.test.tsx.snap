// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoreAction test cases should render correctly 1`] = `
<DocumentFragment>
  <button
    aria-expanded="false"
    aria-haspopup="true"
    aria-label="Dropdown button"
    class="group relative items-center box-border appearance-none select-none whitespace-nowrap font-normal overflow-hidden tap-highlight-transparent data-[pressed=true]:scale-[0.97] outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 px-4 min-w-20 h-10 text-small gap-2 [&>svg]:max-w-[theme(spacing.8)] transition-transform-colors-opacity motion-reduce:transition-none data-[hover=true]:opacity-hover z-10 aria-expanded:scale-[0.97] aria-expanded:opacity-70 subpixel-antialiased text-primary-100 rounded-lg bg-background-100 border-1 data-[focus=true]:border-primary data-[focus=true]:border-1 flex justify-start capitalize"
    data-slot="trigger"
    id="react-aria-:r0:"
    type="button"
  >
    <div
      class="text-primary-100 max-h-6 max-w-6"
    />
  </button>
</DocumentFragment>
`;
